stackcopy_gui_bootstrap.py

```
@@ -0,0 +1,137 @@
+
+            ctypes.windll.user32.MessageBoxW(None, message, title, 0x10)
+            return True
+
+        if sys.platform == "darwin" and Path("/usr/bin/osascript").is_file():
+            escaped_title = title.replace("\\", "\\\\").replace('"', '\\"')
+            escaped_message = message.replace("\\", "\\\\").replace('"', '\\"')
+            script = (
+                f'display alert "{escaped_title}" message "{escaped_message}" '
+                "as critical"
+            )
+            subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)